notebooks/en/enterprise_cookbook_argilla.ipynb

```diff
@@ -0,0 +1,1404 @@
+{
```

Comment:
<p>"If you want, you can also run Argilla locally via Argilla's Docker containers"</p><p> </p><p>"While creating the Argilla Space, it is important to enable persistent storage for your Space to make sure that all annotations are saved."</p><p> </p><p>"You can set their respective names, API keys and passwords for logging into the Argilla interface manually while creating the Space"</p><p> </p><p>I think showing the default values in a table would be a little easier to read.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>